samples/nextjs/scripts/scaffold-component.ts

```diff
@@ -75,13 +75,13 @@ function scaffoldComponent() {
 
   const componentTemplate = `import { Text, Field } from '@sitecore-jss/sitecore-jss-nextjs';
 
-interface IProps {
+interface ${exportVarName}Props {
```

Comment:
This should now be `type` to follow our components. 
Maybe also consider adding a default intersection with `StyleguideComponentProps`?